src/app/common/hooks/use-key-actions.ts

```diff
@@ -55,6 +57,8 @@ export function useKeyActions() {
 
       async signOut() {
         await clearWalletSession();
+        networksActions.changeNetwork('mainnet');
```

Comment:
Wonder if we should handle this at the reducer-layer, the same way we do for bitcoinKeysSlice on L63?

Reply:
We already have a network slice, just saying that in that slice, we could either make a new signout action, or even use the same signout event to reset the state.